ipfs-upload/upload.ts

```diff
@@ -0,0 +1,60 @@
+import { NFTStorage, File } from 'nft.storage';
```

Comment:
We need to figure out how to do this without using this library, as it has over 200 transitive dependencies.